store/datastore/utils.go

```diff
@@ -38,34 +39,33 @@ func rebind(query string) string {
 
 // helper function that converts a simple repsitory list
 // to a sql IN statment.
-func toList(listof []*model.RepoLite) (string, []interface{}) {
-	var size = len(listof)
-	if size > 999 {
-		size = 999
-		listof = listof[:999]
-	}
-	var qs = make([]string, size, size)
-	var in = make([]interface{}, size, size)
-	for i, repo := range listof {
-		qs[i] = "?"
-		in[i] = repo.FullName
-	}
-	return strings.Join(qs, ","), in
-}
+func toList(listof []*model.RepoLite) ([]string, [][]interface{}) {
+	var total = len(listof)
+	var limit = 999
+	var pagesCount = int(math.Ceil(float64(total) / float64(limit)))
```

Comment:
I'm not sure, because `10 / 3 = 3` but `10.0 / 3.0 = 3.3333`... with ceiling to 4 pages.